core/sdk-core/src/main/java/software/amazon/awssdk/core/client/builder/SdkDefaultClientBuilder.java

```diff
@@ -283,8 +289,6 @@ private SdkClientConfiguration mergeGlobalDefaults(SdkClientConfiguration config
                                                   .lazyOption(PROFILE_FILE, conf -> conf.get(PROFILE_FILE_SUPPLIER).get())
                                                   .option(PROFILE_NAME,
                                                           ProfileFileSystemSetting.AWS_PROFILE.getStringValueOrThrow())
-                                                  .option(USER_AGENT_PREFIX, SdkUserAgent.create().userAgent())
-                                                  .option(USER_AGENT_SUFFIX, "")
```

Comment:
Can't see the motivation for this setup. We should try to figure out if this is a safe change.

Reply:
I'd feel more comfortable just keeping these values here. They're public so it's entirely possible some customers are reading or modifying these values.

Reply:
It's frustrating to not be able to remove cruft, but I'm willing to put it back for the sake of less risk affecting edge cases. 

I do want to scrap the definition of `USER_AGENT_PREFIX, SdkUserAgent.create().userAgent()` and set it to the empty string like the suffix. It doesn't make sense to put a huge chunk of text in this param that we know we won't use in any case.

Reply:
Okay, I think leaving it as empty string should be fine.